src/issue-priority.html

```diff
@@ -1,27 +1,39 @@
 <link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/iron-icon/iron-icon.html">
+<link rel="import" href="../bower_components/iron-icons/image-icons.html">
 
 <dom-module id="issue-priority">
   <template>
     <style>
       :host {
-        display: block;
-
-        padding: 10px 20px;
+        display: flex;
+      }
+      :host .icon {
+        color: #a8a8a8;
       }
     </style>
 
+    <template is="dom-repeat" items="[[icons]]">
+      <span class="icon">▊</span>
+    </template>
+
   </template>
 
   <script>
     class IssuePriority extends Polymer.Element {
       static get is() { return 'issue-priority'; }
       static get properties() {
         return {
-          // This shouldn't be neccessary, but the Analyzer isn't picking up
-          // Polymer.Element#rootPath
-          rootPath: String,
+          priority: {
+            type: Number,
+            notify: true,
```

Comment:
As a general tip about using `notify` property, it should be set to `true` only when a value of a property can change *within* the element – either because of a user action or otherwise. It's meant to notify outside that the value has changed. 

In this case, the value for `priority` is always set from the outside, so there is no need for `notify: true`.